### PR TITLE
update airframe reference and add Quad H

### PR DESCRIPTION
**Pull request by bkueng**
Firmware PR: https://github.com/PX4/Firmware/pull/8143
@hamishwillee

### CODE REVIEW

File: en/airframes/airframe_reference.md [hunk 1]
```diff
@@ -401,6 +406,25 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+### Quadrotor h
```
Comment (hamishwillee) [inline en/airframes/airframe_reference.md]: This is consistent, but should it be capital "H" reflecting the shape?

File: en/airframes/airframe_reference.md [hunk 2]
```diff
@@ -401,6 +406,25 @@ This page lists all supported airframes and types including
...
+<img src="../../assets/airframes/types/QuadRotorH.svg" width="29%" style="max-height: 180px;"/>
+</div>
+
+<table style="width: 100%; table-layout:fixed; font-size:1.5rem;">
+ <colgroup><col style="width: 30%"><col style="width: 70%"></colgroup>
+ <thead>
+   <tr><th>Name</th><th></th></tr>
+ </thead>
+<tbody>
+<tr id="copter_quadrotor_h_reaper_500_quad">
```
Comment (hamishwillee) [inline en/airframes/airframe_reference.md]: There are no "common" outputs here. That is correct based on the [config file](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/4040_reaper), but given this is the only case, should we perhaps update that init file and regenerate?

File: en/airframes/airframe_reference.md (hunk 1 above)
Comment (bkueng) [reply en/airframes/airframe_reference.md]: I was going for consistency, but you're right, H makes more sense.

File: en/airframes/airframe_reference.md (hunk 2 above)
Comment (bkueng) [reply en/airframes/airframe_reference.md]: Yep, I added the outputs

**Comment by hamishwillee** [comment]
Great. I have approved the other request. Will check this again once you've merged in the updates.

**Comment by hamishwillee** [comment]
Thanks @bkueng !

**Pull request merged**